root/label/Header.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * @flow
+ * Copyright (C) 2017 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+const React = require('react');
+const {l} = require('../static/scripts/common/i18n');
+const Header = require('../components/Header');
+
+type Props = {
+  label: EntityT;
```

Comment:
`CoreEntityT` would be more accurate. Even better, add a `LabelT` (can just be an alias for `CoreEntityT`). Then we won't have to fix this later.